enhancements/openshift-route-admission-policy.md

```diff
@@ -51,23 +53,19 @@ See the [section][#route-admission-policies]
 
 ## Proposal
 
-Modify the existing `operator.openshift.io`'s custom resource [`IngressController`][ingress-controller-api] to include an enum field `RouteAdmissionPolicy` that can be set to `AllowInterNamespaceClaims` or `Strict`, where `Strict` is default.
-
+Add a new _RouteAdmissionPolicy_ field to the [IngressController API resource][https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go] which allows administrators to configure how cross-namespace hostname claims should be handled by the IngressConroller.
 
 ### User Stories
 
-#### Default Route Admission Policy
-
-The default behavior is `Strict` which match the current behavior and is a secure default.
-
-
 ### Implementation Details/Notes/Constraints [optional]
 
-The openshift-ingress-operator will be able to use the enum `RouteAdmissionPolicy` to enable the environment variable `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK` in the openshift-ingress router. This is enabled when `RouteAdmissionPolicy` is set to `AllowInterNamespaceClaims`.
+The [Ingress Operator](https://github.com/openshift/cluster-ingress-operator) will use `RouteAdmissionPolicy` to configure the environment variable `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK` in the [OpenShift Router](https://github.com/openshift/router).
+
+The default behavior should be to disallow inter-namespace claims, which is secure and backwards compatible with existing IngressController resources.
 
 ### Risks and Mitigations
 
-Addition of the new field requires no migration, as the default matches the current configuration.
+Addition of the new field requires no migration, as the default is not changing.
```

Comment:
Will there be an issue with downgrades?  For routes created after a downgrade, we would fail safe (i.e., prohibit them).  What about routes admitted before a downgrade?

Reply:
Probably worth testing — my first guess is the router will displace the newest one.